libc/config/linux/aarch64/entrypoints.txt

```diff
@@ -494,11 +500,11 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.rintf
     libc.src.math.rintl
     libc.src.math.round
-    libc.src.math.roundf
-    libc.src.math.roundl
     libc.src.math.roundeven
     libc.src.math.roundevenf
     libc.src.math.roundevenl
+    libc.src.math.roundf
+    libc.src.math.roundl
     libc.src.math.scalbn
```

Comment:
Do we just sort all entrypoints with `sort`, or do we sort prefixes and then group type variants of the same function together? I guess the former is easier.